src/code42cli/cmds/securitydata.py

```diff
@@ -234,9 +225,79 @@ def show(state, search_id):
     echo(pformat(response["searches"]))
 
 
+@security_data.command()
+@file_event_options
+@search_options
+@click.option(
+    "--or-query", is_flag=True, cls=searchopt.AdvancedQueryAndSavedSearchIncompatible
+)
+@sdk_options()
+@server_options
+def send_to(
+    state,
+    format,
+    hostname,
+    protocol,
+    begin,
+    end,
+    advanced_query,
+    use_checkpoint,
+    saved_search,
+    or_query,
+    **kwargs
+):
+    """Send alerts to the given server address."""
+    output_logger = logger_factory.get_server_logger(hostname, protocol, format)
+    _extract_events(
+        state,
+        output_logger,
+        begin,
+        end,
+        advanced_query,
+        use_checkpoint,
+        saved_search,
+        or_query,
+        **kwargs,
+    )
+
+
 def _get_file_event_extractor(sdk, handlers):
     return FileEventExtractor(sdk, handlers)
 
 
 def _get_file_event_cursor_store(profile_name):
     return FileEventCursorStore(profile_name)
+
+
+def _extract_events(
+    state,
+    output_logger,
+    begin,
+    end,
+    advanced_query,
+    use_checkpoint,
+    saved_search,
+    or_query,
+    **kwargs
+):
+    cursor = (
+        _get_file_event_cursor_store(state.profile.name) if use_checkpoint else None
+    )
+    handlers = ext.create_handlers(
+        state.sdk, FileEventExtractor, output_logger, cursor, use_checkpoint
+    )
+    extractor = _get_file_event_extractor(state.sdk, handlers)
+    extractor.use_or_query = or_query
+    extractor.or_query_exempt_filters.append(f.ExposureType.exists())
+    if advanced_query:
+        extractor.extract_advanced(advanced_query)
+    elif saved_search:
+        extractor.extract(*saved_search._filter_group_list)
+    else:
+        if begin or end:
+            state.search_filters.append(
+                ext.create_time_range_filter(f.EventTimestamp, begin, end)
+            )
+        extractor.extract(*state.search_filters)
+    if handlers.TOTAL_EVENTS == 0 and not errors.ERRORED:
```

Comment:
could simply to `not handlers.TOTAL_EVENTS`

Reply:
`if not handlers.TOTAL_EVENTS and not errors.ERRORED`? missing `not`